src/App.tsx

```diff
@@ -1,17 +1,95 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import classNames from 'classnames';
+import { TodoList } from './components/TodoList/TodoList';
 
 import users from './api/users';
+import todosFromServer from './api/todos';
+import { UserInfo } from './components/UserInfo/UserInfo';
 
 const App: React.FC = () => {
+  const [todos, setTodos] = useState(todosFromServer);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState('0');
+  const [inputError, setInputError] = useState(false);
+  const [selectError, setSelectError] = useState(false);
+  const userInfo = users.find(user => user.id === +userId);
+
+  const handlerSubmit = (event: { preventDefault: () => void; }) => {
```

Comment:
```suggestion
  const handlerSubmit = (event: { preventDefault: () => void; }) => {
```
use provided react type